@here/harp-mapview/lib/MapViewTaskScheduler.ts

```diff
@@ -77,6 +77,10 @@ export class MapViewTaskScheduler extends THREE.EventDispatcher {
         let numItemsLeft = this.taskQueue.numItemsLeft();
         currentFrameEvent?.setValue("TaskScheduler.numPendingTasks", numItemsLeft);
 
+        // Needed for tests that dispose but that still have running tasks after MapView disposed.
```

Comment:
Otherwise the tasks are executed on a mapView that has disposed and the update method sends long of messages to the console.